package.json

```diff
@@ -113,7 +113,8 @@
     "prepare": "husky",
     "prebuild": "node ./build/check-node-version.js",
     "pretest": "node ./build/check-node-version.js",
-    "postbuild": "prettier --write ./locales/_template.json ./doc/rule-descriptions.md"
+    "postbuild": "prettier --write ./locales/_template.json ./doc/rule-descriptions.md",
+    "postinstall": "patch-package"
```

Comment:
We really want to avoid adding a postinstall script to axe-core - this would add it as a step that axe-core consumers run when doing `npm install axe-core`, which we don't want. Instead, we'll want to run the command line as a build step. This will likely mean adding a grunt task wrapping it to `/build/tasks/` that you reference from `Gruntfile.js`.